src/atoms/price/Price.html

```diff
@@ -1,24 +1,28 @@
-<div class="price">
-  <!-- @slot Custom old price slots (Named slots) -->
+<div class="a-price">
+  <!-- @slot Custom old price (Named slots) -->
```

Comment:
imo we can transform it to scoped slot and pass needed data, for example like `:special-price`, `:regular-price`. Please let me know what do you think, we can replace all named slots in this component to scoped.

Reply:
as we discussed through the zoom